modules/distribution/src/repository/resources/conf/unit-resolve.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Remove separation new lines. JSON we don't keep new lines.

Reply:
Refactored in master